runtime/onert/api/src/nnfw_api_internal.h

```diff
@@ -168,7 +168,6 @@ struct nnfw_session
   State _state{State::INITIALIZED};
   std::shared_ptr<onert::ir::Subgraphs> _subgraphs;
   std::unique_ptr<onert::compiler::CompilerOptions> _coptions;
-  std::unique_ptr<onert::compiler::Compiler> _compiler;
```

Comment:
It is the main change of this PR. `_compiler` is only alive during `compile()` though `CompilationResult` (#9280) may live after `compile()`.